src/main/java/io/redlink/more/data/repository/StudyRepository.java

```diff
@@ -148,6 +148,14 @@ SELECT start, participant_id, alias, COALESCE(sg.duration, s.duration) AS durati
     private static final String GET_PARTICIPANT_OBSERVATION_PROPERTIES_BY_KEY_EXISTS =
             "SELECT * FROM participant_observation_properties WHERE participant_id = :participant_id AND study_id = :study_id AND properties ?? :key";
 
+    private static final String GET_ALL_PARTICIPANT_OBSERVATION_PROPERTIES_BY_OBSERVATION_TYPE =
+            """
+                    SELECT p.*
+                    FROM participant_observation_properties AS p
+                    JOIN observations AS o
```

Comment:
Ich hatte sofort SELECT participant.* gelesen ...

```suggestion
                    SELECT pop.*
                    FROM participant_observation_properties AS pop
                    JOIN observations AS o
```